app/src/main/java/org/jellyfin/androidtv/util/DeviceUtils.kt

```diff
@@ -26,6 +26,9 @@ object DeviceUtils {
 	// Nvidia Shield TV Model
 	private const val SHIELD_TV_MODEL = "SHIELD Android TV"
 
+	// Google Chromecast HD Model
+	private const val GOOGLE_CHROMECAST_HD_MODEL = "Chromecast HD" // Reports to support 4K - Cannot downscale in device in realtime
```

Comment:
```suggestion
	private const val GOOGLE_CHROMECAST_HD_MODEL = "Chromecast HD"
```

These are just constants for names, we don't specify why they exist. That's on the usage part.